CHANGELOG.md

```diff
@@ -15,6 +15,9 @@
 - Simplified and clarified the removal policy of deprecated API elements
   to follow semantic versioning 2.0.0. For more information, please refer to
   [this GitHub discussion](https://github.com/firecracker-microvm/firecracker/discussions/4135).
+- [#4180](https://github.com/firecracker-microvm/firecracker/pull/4180):
+  Refactored error propagation to avoid logging and printing an error on
+  exits with a zero exit code.
```

Comment:
```suggestion
  exits with a zero exit code. Now on successful exit, Firecracker prints
 "Firecracker exited successfully" instead of
 "RunWithoutApiError error: MicroVMStopped without an error: Ok"
```

Reply:
I think

> Now on successful exit, Firecracker prints "Firecracker exited successfully"

can you be included but 

> instead of "RunWithoutApiError error: MicroVMStopped without an error: Ok"

should not be included since this is only 1 of many thing it can print when exiting successfully, by including this it gives the impression the other successful exit cases are unchanged and won't print  "Firecracker exited successfully".